# ChIP Seq Analysis Papers

<ul>

                             

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(1).pdf" style="text-decoration:none;">A rapid and robust method for single cell chromatin accessibility profiling</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(2).pdf" style="text-decoration:none;">An efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(3).pdf" style="text-decoration:none;">A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(4).pdf" style="text-decoration:none;">The Gene Ontology Handbook</a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(5).pdf" style="text-decoration:none;">RSAT matrix-clustering: dynamic exploration and redundancy reduction of transcription factor binding motif collections</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(6).pdf" style="text-decoration:none;">Characterizing protein-DNA binding event subtypes in ChIP-exo data</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(7).pdf" style="text-decoration:none;">ATAC‑seq normalization method can
significantly affect differential accessibility analysis and interpretation</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(8).pdf" style="text-decoration:none;"> CoBATCH for high-throughput single-cell epigenomic profiling </a></li>
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(9).pdf" style="text-decoration:none;">Simplified ChIP-exo assays</a></li>
  
   
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(10).pdf" style="text-decoration:none;">Global Prediction of Chromatin Accessibility Using RNA-seq from Small Number of Cells </a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(11).pdf" style="text-decoration:none;">Self-reporting transposons enable simultaneous readout of gene expression and transcription factor binding in single cells</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(12).pdf" style="text-decoration:none;">Index and biological spectrum of accessible DNA elements in the human genome</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(13).pdf" style="text-decoration:none;">Enhancers and super-enhancers have an equivalent regulatory role in embryonic stem cells through regulation of single or multiple genes</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(14).pdf" style="text-decoration:none;">Recurrent promoter mutations in melanoma are defined by an extended context-specific mutational signature</a></li>
                              
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(15).pdf" style="text-decoration:none;">Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(16).pdf" style="text-decoration:none;">Comparison of differential
accessibility analysis strategies for ATAC-seq data</a></li>

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(17).pdf" style="text-decoration:none;">chromVAR: Inferring transcription factor variation from single-cell epigenomic data</a></li>   
  
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(18).pdf" style="text-decoration:none;">Reproducible inference of transcription factor footprints in ATAC-seq and DNase-seq datasets via protocol-specific bias modeling</a></li> 

  
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(19).pdf" style="text-decoration:none;">Mapping Histone Modifications in Low Cell Number and Single Cells Using Antibody guided Chromatin Tagmentation (ACT-seq)</a></li> 

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(20).pdf" style="text-decoration:none;">Wellington-bootstrap: differential DNase-seq footprinting identifies cell-type determining transcription factors</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(21).pdf" style="text-decoration:none;">Extensive Divergence of Transcription Factor Binding in Drosophila Embryos with Highly Conserved Gene Expression</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(22).pdf" style="text-decoration:none;">Single cell chromatin immunocleavage sequencing (scChIC-Seq) to profile histone modification</a></li> 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(23).pdf" style="text-decoration:none;">Genome-wide identification and characterization of HOT regions in the human genome</a></li> 
 

   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(24).pdf" style="text-decoration:none;">A curated benchmark of enhancer-gene interactions for evaluating enhancer-target gene prediction methods</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(25).pdf" style="text-decoration:none;">Ultra-parallel ChIP-seq by barcoding of intact nuclei</a></li>                              
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(26).pdf" style="text-decoration:none;">Computational Identification of Genomic Features That Influence 3D Chromatin Domain Formation</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(27).pdf" style="text-decoration:none;">Which Genetics Variants in DNase-Seq Footprints Are More Likely to Alter Binding?</a></li>
   
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(28).pdf" style="text-decoration:none;">CUT&Tag for efficient epigenomic profiling of small samples and single cells</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(29).pdf" style="text-decoration:none;">Genetic dissection of the α-globin super-enhancer in vivo </a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(30).pdf" style="text-decoration:none;">Multiplex enhancer-reporter assays uncover unsophisticated TP53 enhancer logic</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(31).pdf" style="text-decoration:none;">ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia</a></li> 
    <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(32).pdf" style="text-decoration:none;">Mutational Biases Drive Elevated Rates of Substitution at Regulatory Sites across Cancer Types</a></li> 

   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(33).pdf" style="text-decoration:none;">Dissecting neural differentiation regulatory networks through epigenetic footprinting</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(34).pdf" style="text-decoration:none;">Systematic comparison of monoclonal versus polyclonal antibodies for mapping histone modifications by ChIP-seq</a></li> 
 
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(35).pdf" style="text-decoration:none;">Active promoters give rise to false positive 'Phantom Peaks' in ChIP-seq experiments</a></li> 

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(36).pdf" style="text-decoration:none;">A User's Guide to the Encyclopedia of DNA Elements (ENCODE)</a></li> 
 
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(37).pdf" style="text-decoration:none;">Identification of Transcription Factor Binding Sites using ATAC-seq</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(38).pdf" style="text-decoration:none;">Simultaneous quantification of protein-DNA contacts and transcriptomes in single cells</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(39).pdf" style="text-decoration:none;">ChIP-seq and Beyond: new and improved methodologies to detect and characterize protein-DNA interactions</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(40).pdf" style="text-decoration:none;">Identifying differential transcription factor binding in ChIP-seq</a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(41).pdf" style="text-decoration:none;">A toolbox of immunoprecipitation-grade monoclonal antibodies to human transcription factors</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(42).pdf" style="text-decoration:none;">Large-scale identification of sequence variants impacting human transcription factor occupancy in vivo</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(43).pdf" style="text-decoration:none;">Analysis of optimized DNase-seq reveals intrinsic bias in transcription factor footprint identification</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(44).pdf" style="text-decoration:none;">Systematic evaluation of factors influencing ChIP-seq fidelity</a></li>
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(45).pdf" style="text-decoration:none;">A comprehensive comparison of tools for differential ChIP-seq analysis</a></li>  
   
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(46).pdf" style="text-decoration:none;">Transcription Factors Bind Thousands
of Active and Inactive Regions in the Drosophila Blastoderm</a></li> 
                             
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(47).pdf" style="text-decoration:none;">Survey of variation in human
transcription factors reveals prevalent DNA binding changes</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(48).pdf" style="text-decoration:none;">BIDCHIPS: bias decomposition
and removal from ChIP‑seq data clarifies true binding signal and its functional correlates</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(49).pdf" style="text-decoration:none;">Accounting for GC-content bias reduces systematic errors and batch effects in ChIP-Seq peak callers</a></li>
                              
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(50).pdf" style="text-decoration:none;">Accurate promoter and enhancer identification in 127 ENCODE and Roadmap Epigenomics cell types and tissues by GenoSTAN</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(51).pdf" style="text-decoration:none;">Accurate inference of transcription factor binding from DNA sequence and chromatin accessibility data</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(52).pdf" style="text-decoration:none;">Computational Tools for Stem Cell Biology</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(53).pdf" style="text-decoration:none;">From reads to insight: a hitchhiker’s guide to ATAC-seq data analysis</a></li>
 
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(54).pdf" style="text-decoration:none;">Enhancer-promoter interactions are encoded by complex genomic signatures on looping chromatin </a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(55).pdf" style="text-decoration:none;">MixChIP: a probabilistic method for cell type specific protein-DNA binding analysis</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(56).pdf" style="text-decoration:none;">Transcription factor motifs </a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(57).pdf" style="text-decoration:none;">High-throughput ChIPmentation: freely scalable, single day ChIPseq data generation from very low cell-numbers</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(58).pdf" style="text-decoration:none;">Detecting broad domains and narrow peaks in ChIP-seq data with hiddenDomains</a></li>
    <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(59).pdf" style="text-decoration:none;">High resolution discovery of chromatin interactions</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(60).pdf" style="text-decoration:none;">10 Characterization of network topology </a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(61).pdf" style="text-decoration:none;">Machine learning approaches to genomics</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(62).pdf" style="text-decoration:none;">SNPsplit: Allele-specific splitting of alignments between genomes with known SNP genotypes [version 2; peer review: 3 approved]</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(63).pdf" style="text-decoration:none;">Systematic identification of protein combinations mediating chromatin looping</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(64).pdf" style="text-decoration:none;">A computational pipeline for comparative ChIP-seq analyses</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(65).pdf" style="text-decoration:none;">Profiling of pluripotency factors in individual stem cells and early embryos </a></li> 

   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(66).pdf" style="text-decoration:none;">The Functional Consequences of Variation in Transcription Factor Binding</a></li> 
 
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(67).pdf" style="text-decoration:none;">Single-cell ChIP-seq reveals cell subpopulations defined by chromatin state</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(68).pdf" style="text-decoration:none;">Recent advances in ChIP-seq analysis: from quality management to whole-genome annotation</a></li> 
 
  
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(69).pdf" style="text-decoration:none;">12 Impact of functional information on understanding variation</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(70).pdf" style="text-decoration:none;">4 RNA and chromatin modification patterns around promoters</a></li> 
  
 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(71).pdf" style="text-decoration:none;">Spatial Clustering for Identification of ChIP-Enriched Regions (SICER) to Map Regions of Histone Methylation Patterns in Embryonic Stem Cells</a></li>
 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(72).pdf" style="text-decoration:none;">Calling Cards enable multiplexed identification of the genomic targets of DNA-binding proteins</a></li> 
 
 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(73).pdf" style="text-decoration:none;">MAPS: model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(74).pdf" style="text-decoration:none;">Beyond library size: a field guide to NGS normalization</a></li>
    <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(75).pdf" style="text-decoration:none;">Hierarchy within the mammary STAT5-driven Wap super-enhancer</a></li>                        
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(76).pdf" style="text-decoration:none;">Unraveling the 3D genome: genomics tools for multi-scale exploration</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(77).pdf" style="text-decoration:none;">Identifying ChIP-seq enrichment using MACS</a></li> 
 
 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(78).pdf" style="text-decoration:none;">Three-dimensional disorganization of the cancer genome occurs coincident with long-range genetic and epigenetic alterations</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(79).pdf" style="text-decoration:none;">Estimating the functional impact of INDELs in transcription factor binding sites: a genome-wide landscape</a></li>


 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(80).pdf" style="text-decoration:none;">2 Chromatin patterns at transcription factor binding sites</a></li> 
 
 
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(81).pdf" style="text-decoration:none;">7 DNA methylation</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(82).pdf" style="text-decoration:none;">ChIP-Seq: advantages and challenges of a maturing technology</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(83).pdf" style="text-decoration:none;">Features that define the best ChIP-seq peak calling algorithms</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(84).pdf" style="text-decoration:none;">A practical guide for DNase-seq data analysis: from data management to common applications</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(85).pdf" style="text-decoration:none;">8 Enhancer discovery and characterization</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(86).pdf" style="text-decoration:none;">Three-dimensional genome architecture and emerging technologies: looping in disease</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(87).pdf" style="text-decoration:none;">3 Characterization of intergenic regions and gene definition</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(88).pdf" style="text-decoration:none;">Protein binding and methylation on looping chromatin accurately predict distal regulatory interactions</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(89).pdf" style="text-decoration:none;">13 Impact of evolutionary selection on functional regions</a></li>
  
  
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(90).pdf" style="text-decoration:none;"> Histone Modifications: Insights
into Their Influence on Gene Expression</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(91).pdf" style="text-decoration:none;">HiChIP: Efficient and sensitive analysis of protein-directed genome architecture</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(92).pdf" style="text-decoration:none;">Panoramic views of
the early epigenome</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(93).pdf" style="text-decoration:none;"> 5 Epigenetic regulation of RNA processing</a></li>
  <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(94).pdf" style="text-decoration:none;">Exploring long-range genome interaction data using the WashU Epigenome Browser</a></li> 
  
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(95).pdf" style="text-decoration:none;">Practical Guidelines for the Comprehensive Analysis of ChIP-seq Data</a></li>  
  
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(96).pdf" style="text-decoration:none;">A review of ensemble methods for de novo motif discovery in ChIP-Seq data</a></li> 
  
  
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(97).pdf" style="text-decoration:none;">6 Non-coding RNA characterization</a></li>


 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(98).pdf" style="text-decoration:none;">BroadPeak: a novel algorithm for identifying broad peaks in diffuse ChIP-seq datasets</a></li> 
  
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(99).pdf" style="text-decoration:none;">MCAST: scanning for cis-regulatory motif clusters</a></li>  
  
<li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(100).pdf" style="text-decoration:none;">epic2 efficiently finds diffuse domains in ChIP-seq data</a></li>  
  
 <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(101).pdf" style="text-decoration:none;">Integrative analysis of 10,000 epigenomic maps across 800 samples for regulatory genomics and disease dissection</a></li> 
  
   <li><a target="_blank" href="https://github.com/manjunath5496/ChIP-Seq-Analysis-Papers/blob/master/c(102).pdf" style="text-decoration:none;">Genome-wide footprinting: Ready for prime time?</a></li> 
  
   
 </ul>
   
   
